src/services/userCompanyAssociationService.ts

```diff
@@ -217,6 +209,32 @@ export const addUserEmailAssociation = async (email: string, companyNumber: stri
         }
     }
 };
+export const getCompanyAssociations = async (companyNumber: string, cancellationOrRemoval: Cancellation | Removal | undefined): Promise<Associations> => {
+    // We will replace this hard coded value with the API call once the API is available
+
+    if (cancellationOrRemoval && "cancelPerson" in cancellationOrRemoval) {
+        return getCompanyAssociationsAfterCancellation(companyNumber, cancellationOrRemoval);
```

Comment:
technically they both do the same thing. If cancelled or removed the status is set to Removed. Dont know if we need 2 separate calls here?

Reply:
similar as above